src/app_engine/probers.py

```diff
@@ -155,23 +187,23 @@ def create_collider_active_host(self, old_active_host, probing_results):
 
   def get(self):
     results = {}
-    for collider_instance in constants.WSS_HOST_PORT_PAIRS:
-      results[collider_instance] = self.probe_collider_instance(
-          collider_instance)
+    for instance in constants.WSS_INSTANCES:
```

Comment:
Why did this change happen here? it doesn't seem to make sense in this context. I mean, the change makes sense separately.